src/array-to-integer.js

```diff
@@ -0,0 +1,59 @@
+const _ = require('lodash');
+
+const fromIntegerArray = function(integerArray) {
+  return getValue(_.uniq(integerArray), 'number', (function(_this) {
+    return function(bitmap, value, index) {
```

Comment:
You can use `=>` here
```suggestion
  return getValue(_.uniq(integerArray), 'number', (bitmap, value, index) => {
```